sw/airborne/boards/px4fmu_4.0/baro_board.h

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
This file should actually not be needed if the board is simply set to px4fmu in the makefile as commented above.